test/components/developers/card_component_test.rb

```diff
@@ -58,7 +58,7 @@ class CardComponentTest < ViewComponent::TestCase
     test "doesn't render recently added badge if developer was added more than 7 days ago" do
       @developer.update!(created_at: 2.weeks.ago)
       render_inline(CardComponent.new(developer: @developer))
-      assert_no_text I18n.t("developers.card_component.recently_added")
+      assert_no_text I18n.t("developers.badges_component.recently_added")
```

Comment:
For some reason this only failed when I upgraded Rails even though it appears that it should have failed before.